examples/asr/speech_to_text_rnnt.py

```diff
@@ -79,15 +59,86 @@
     +model.optim.args.betas=[0.8,0.5]\
     +model.optim.args.weight_decay=0.0005
 
+Finetune a model
+1) Finetune from a .nemo file
+
+```sh
+    python examples/asr/speech_to_text_rnnt.py \
+        --config-path=<path to dir of configs> \
+        --config-name=<name of config without .yaml>) \
+        model.train_ds.manifest_filepath="<path to manifest file>" \
+        model.validation_ds.manifest_filepath="<path to manifest file>" \
+        trainer.gpus=-1 \
+        trainer.max_epochs=50 \
+        +init_from_nemo_model="<path to .nemo model file>"
+```
+
+2) Finetune from a pretrained model (via NGC)
+
+```sh
+    python examples/asr/speech_to_text_rnnt.py \
+        --config-path=<path to dir of configs> \
+        --config-name=<name of config without .yaml>) \
+        model.train_ds.manifest_filepath="<path to manifest file>" \
+        model.validation_ds.manifest_filepath="<path to manifest file>" \
+        trainer.gpus=-1 \
+        trainer.max_epochs=50 \
+        +init_from_pretrained_model="<name of pretrained checkpoint>"
+```
 """
 
+import pytorch_lightning as pl
+from omegaconf import OmegaConf, open_dict
+
+from nemo.collections.asr.models import EncDecRNNTModel
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+from nemo.utils.exp_manager import exp_manager
+
+
+def restore_weights_if_required(model: EncDecRNNTModel, cfg: OmegaConf):
```

Comment:
We are duplicating mostly the same method 8 times in 4 scripts. Let's merge them as much as possible and put it into a common place to be used by all. Maintaining the same method scattered like this can be a burden.